prow/plugins/trigger/trigger.go

```diff
@@ -260,3 +267,43 @@ func skipRequested(c Client, pr *github.PullRequest, skippedJobs []config.Presub
 	}
 	return errorutil.NewAggregate(errors...)
 }
+
+func createProwJobRefs(c Client, pr *github.PullRequest) (prowapi.Refs, error) {
```

Comment:
It would be better to update `pjutil.NewPresubmit` to do this, instead of having it here.

Reply:
Thanks for the review. This was added as `pjutil` does not currently have access to a github client, neither does `github.PullRequest` include PR's changes (matching github's API). Would it be preferable for `NewPresubmit` to allow an optional list of changed files instead? or does adding a github client requirement seem like a better option?

Reply:
Let's add the list of files as a parameter to the `pjutil` call -- still unclear what the best way is to get the list of files.

Reply:
Thanks for the update. Done.